File: python/agentic_commerce_protocol/models/event_data_order.py
# this file is @generated
import typing as t

from .common import BaseModel
from .refund import Refund
from .webhook_order_status import WebhookOrderStatus


class EventDataOrder(BaseModel):
    type: str
    checkout_session_id: str
    permalink_url: str
    status: WebhookOrderStatus
    refunds: t.List[Refund]

File: kotlin/src/main/kotlin/com/svix/agentic_commerce_protocol/EventDataOrder.kt
// This file is @generated
package com.svix.agentic_commerce_protocol

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class EventDataOrder(
    val type: String,
    @SerialName("checkout_session_id") val checkoutSessionId: String,
    @SerialName("permalink_url") val permalinkUrl: String,
    val status: WebhookOrderStatus,
    val refunds: List<Refund>,
)
